hackathon_site/hardware/test_api.py

```diff
@@ -598,7 +627,34 @@ def test_search_filter(self):
         results = data["results"]
 
         returned_ids = [res["id"] for res in results]
-        self.assertCountEqual(returned_ids, [self.order_3.id])
+        self.assertCountEqual(returned_ids, [self.order_3.id, self.order_4.id])
+
+    def test_status_filter_with_only_view_permissions(self):
```

Comment:
We know that filters are applied after calling `get_queryset` - that's something that DRF guarantees for us. We also know that the status filter works (since it's tested above). Hence, there's no need for this test